bugbug/models/duplicate.py

```diff
@@ -32,7 +28,10 @@ def fit(self, X, y):
 
 
 class DuplicateModel(BugCoupleModel):
-    def __init__(self, lemmatization=False):
+    def __init__(self, training_size, lemmatization=False):
```

Comment:
We'll have to set a default here, otherwise scripts/trainer.py will fail. You can use the same default you set in run.py.

Reply:
Done.